tests/test.py

```diff
@@ -0,0 +1,41 @@
+import hydra, os, subprocess
+from omegaconf import DictConfig
+
+@hydra.main(version_base=None, config_name="config")
+def main(config : DictConfig) -> None:
+
+    print("read test config from tests/conf/config.yaml")
```

Comment:
The terminal output should began with a capital letter like the normal sentence.